pyroute2/dhcp/xids.py

```diff
@@ -0,0 +1,75 @@
+import random
+from typing import Any, Optional
+
+from pyroute2.dhcp.fsm import State
+
+
+def random_xid_prefix() -> int:
+    '''A (max) 32 bit random int with its lowest nibble set to 0.
+
+    These last 4 bits are used by the client to store its state, in the form
+    of the associated `State` value, when sending a DHCP message.
+    '''
+    return random.randint(0x00000010, 0xFFFFFFF0)
```

Comment:
Probably it's time to start using `secrets.SystemRandom()`? What would you say?

Reply:
Well it's not really a secret or cryptographic value but it wouldn't hurt ! I'll change it